app/assets/javascripts/filepaths/show.js

```diff
@@ -1,49 +1,46 @@
-// $( document ).ready( function() {
-//     $.ajax({
-//         url: "/api/filepaths/" + $('#filepath_id').text() + "/commits",
-//         dataType: 'json',
-//         success: function(jsonData) {
-//             //link author's email by author_id
-//             jsonData.forEach(function(author, index) {
-//                 jsonData[index]['email'] = authorEmails[author['author_id']];
-//             });
-//
-//             $('#datatable').dataTable({
-//                 destroy: true,
-//                 data: jsonData,
-//                 rowCallback: function(row, data, index){
-//                   $(row).on('click', function(){
-//                     var url = "/commits/" + data.id;
-//                     window.location.assign(url);
-//                   })
-//                 },
-//                 columnDefs: [
-//                   { "width": "10%", "targets": 0 }
-//                 ],
-//                 columns: [
-//                     { title: 'Commit Hash', data: 'commit_hash', defaultContent: 'N ull'},
-//                     { title: 'Author Email', data: 'email', defaultContent: 'Null'},
-//                     { title: 'Date Created', data: 'date_created', defaultContent: 'Null',
-//                       render: function(data) {
-//                         return moment(data).format('MMMM Do YYYY, h:mm a');
-//                       }
-//                     }
-//                 ]
-//             });
-//         }
-//     });
-//
-//     $('#datatable').on('mouseover', 'tbody tr', function() {
-//         $(this).css({
-//            'cursor': 'pointer',
-//             'color': 'blue'
-//         });
-//     })
-//
-//     $('#datatable').on('mouseout', 'tbody tr', function() {
-//         $(this).css({
-//             'cursor': 'default',
-//             'color': 'inherit'
-//         });
-//     })
-// });
+$( document ).ready( function() {
+    $.ajax({
+        url: "/api/filepaths/"+ $('#filepath_id').text() +"/vulnerabilities",
```

Comment:
```suggestion
        url: "/api/filepaths/"+ $('#filepath_id').text() +"/vulnerabilities?short_descriptions=true",
```
This will shorten the descriptions on the database call.

Reply:
Yeah it would be something like that query. You might not need the outer join though because every vulnerability has at least one tag now. If that makes it easier.